scripts/us_census/pep/population_estimates_by_asr/process.py

```diff
@@ -203,61 +239,125 @@ def process(self):
         output_path = os.path.dirname(self._cleaned_csv_file_path)
         if not os.path.exists(output_path):
             os.mkdir(output_path)
+        processed_count = 0
+        total_files_to_process = len(self._input_files)
+        if total_files_to_process is None:
```

Comment:
Length of total_files_to_process will never be None. So please make it as total_files_to_process == 0